src/rpcblockchain.cpp

```diff
@@ -921,7 +921,7 @@ UniValue rainbymagnitude(const UniValue& params, bool fHelp)
 
     UniValue res(UniValue::VOBJ);
 
-    double dAmount = params[0].get_real();
+    double dAmount = RoundFromString(params[0].get_real(), 8);
```

Comment:
If get_real returns float, then RoundFromString will probably not work.

Reply:
`get_real` returns a double.

Reply:
double is float, is double float. you know what i mean

Reply:
@tomasbrod Oh now I see what you mean, double/float is not a string. How does that even compile?

Reply:
It does not.